packages/mantine/src/BlockNoteView.tsx

```diff
@@ -74,24 +74,28 @@ export const BlockNoteView = <
     [defaultColorScheme, theme]
   );
 
+  const mantineCssVariablesRef = useRef<HTMLDivElement>(null);
+
   return (
     <ComponentsContext.Provider value={components}>
       {/* `cssVariablesSelector` scopes Mantine CSS variables to only the editor, */}
       {/* as proposed here:  https://github.com/orgs/mantinedev/discussions/5685 */}
-      <MantineProvider
-        theme={mantineTheme}
-        cssVariablesSelector=".bn-mantine"
-        // This gets the element to set `data-mantine-color-scheme` on. Since we
-        // don't need this attribute (we use our own theming API), we return
-        // undefined here.
-        getRootElement={() => undefined}>
-        <BlockNoteViewRaw
-          className={mergeCSSClasses("bn-mantine", className || "")}
-          theme={typeof theme === "object" ? undefined : theme}
-          {...rest}
-          ref={ref}
-        />
-      </MantineProvider>
+      <div className={"bn-mantine-css-variables"} ref={mantineCssVariablesRef}>
+        <MantineProvider
+          theme={mantineTheme}
+          cssVariablesSelector=".bn-mantine-css-variables"
```

Comment:
From what I understand of their docs, this is just about selector their stylesheet would reference the styles for. So, I don't think that this actually needs the element to exist, as it is only being applied through CSS.

Am I getting this wrong?